sema/commons/j2/j2_functions.py

```
@@ -61,7 +64,7 @@
     return xsd_value(str(content), quote, "xsd:integer")
 
 
-def xsd_format_double(content, quote, suffix):
+def xsd_format_double(content: any, quote: str, suffix: str) -> str:
```

Ruff (0.8.2)

Unused function argument: `suffix`

(ARG001)